packages/api-gateway/package.json

```diff
@@ -31,13 +31,16 @@
     "@fastify/cors": "^8.0.0",
     "@fastify/helmet": "^9.1.0",
     "@fastify/multipart": "^7.1.0",
+    "@fastify/static": "^6.5.0",
     "@fastify/swagger": "^7.4.1",
     "@siiges-services/authentication": "^1.0.0",
     "@siiges-services/core": "^1.0.0",
+    "@siiges-services/files": "^1.0.0",
```

Comment:
I think you didn't add files package in this PR, I think you should have make that package a PR before this, otherwise I won't be able to run the api-gateway